qe-tests/cypress/integration/tests/login.spec.ts

```diff
@@ -0,0 +1,14 @@
+import{ userNameInput, userPasswordInput, LoginButton } from '../views/login.view'
+
+const user_password = Cypress.env('pass');
+
+describe('Log In', () => {
+
+    it('Login to pathfinder', () => {
+        cy.visit("https://tackle-mta-pathfinder.apps.ocp4.prod.psi.redhat.com")
```

Comment:
Can we keep this url as well in some env or constant variable at some top level?